src/commands/open.ts

```
@@ -93,11 +151,15 @@ async function handleMissingPlaceFile(placeFile: string, isCustomPlace: boolean)
 	}
 }
 
-async function ensurePlaceFileExists(placeFile: string, isCustomPlace: boolean): Promise<void> {
+async function ensurePlaceFileExists(
+	placeFile: string,
+	isCustomPlace: boolean,
+	config: ResolvedConfig,
+): Promise<void> {
 	try {
 		await access(placeFile);
```

ESLint

[error] Unsafe call of a type that could not be resolved.

(ts/no-unsafe-call)